docs/source/algorithms/SaveNexusGeometry-v1.rst

```diff
@@ -10,35 +10,60 @@
 Description
 -----------
 
-TODO: Enter a full rst-markup description of your algorithm here.
+Saves the Instrument geometry from the workspace to a Nexus file. Saves detector positions and rotations.
+Current version (1.0) does NOT save instrument shapes of instrument components. file is compliant to Nexus standards.
```

Comment:
Good start, and I think incrementing the version of the algorithm once we have shape functionality is a nice idea too, though if we do it all within one release cycle, we wouldn't have to.